### PR TITLE
Delete week-3.js and add solved tasks for third weeks  

### PR DESCRIPTION
**Pull request by Masha-Mascot-00**
I add solved javascript tasks on the following topics: variables, objects and methods, if/else statement, function and arrays.

### CODE REVIEW

File: masha-mascot/Week_3_JS_tasks/Function and arrays/PartyTimes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  if (checkValue === -1) {
+    return "Delicious";
+  } else {
+    return "I won't come here again.";
+  }
+}
+
+let guestReview = guestCheckedPinapple(arrayCocktail);
+console.log(guestReview);
```
Comment (valeriashpiner) [inline masha-mascot/Week_3_JS_tasks/Function and arrays/PartyTimes.js:17]: ❤️ like

File: masha-mascot/Week_3_JS_tasks/If-Else statement/fashionStoreLoli.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+let numberBlueTshirts = 'N';
+
+numberBlueTshirts != 0
```
Comment (valeriashpiner) [inline masha-mascot/Week_3_JS_tasks/If-Else statement/fashionStoreLoli.js:3]: `numberBlueTshirts !== 0`

https://www.w3schools.com/js/js_comparisons.asp

Comment (Masha-Mascot-00) [reply masha-mascot/Week_3_JS_tasks/If-Else statement/fashionStoreLoli.js:3]: `numberBlueTshirts !== 0`

Comment (Masha-Mascot-00) [reply masha-mascot/Week_3_JS_tasks/If-Else statement/fashionStoreLoli.js:3]: If we change third line like this `numberBlueTshirts !== 0` , the program will not work correctly with the following values:
`let numberBlueTshirts = '';`
`let numberBlueTshirts = ' ';`

**Pull request merged**